src/app/help/en/welcome.md

```diff
@@ -221,87 +220,88 @@ You can also exchange your STEEM and SBD for bitcoin on a third-party exchange s
 <a href="#Table_of_Contents">^</a>
 ### <span id="Plagiarism">Plagiarism</span>
 
-The community is looking for you to add your own personal touch to your articles.
+The community is looking for you to create engaging content. 
 
-Plagiarizing, that is posting someone else's work as if it were your own, is very frowned upon by the Steemit community.
+Plagiarising, which is posting someone else's material as if it were your own, is very frowned upon by the Steem community.
 
-If you are using anyone else's material as part of your posts (including images) - please cite your sources.
+If you are using anyone else's material as part of your posts (including images), even if it is free to use, please cite your sources. You should make sure that you are not violating any copyright laws if you are using someone else's material/images. 
 
-Also, make sure that you are not violating any copyright laws if you are using someone else's material/images. Limited, sourced material sharing is OK under fair use and fair dealing doctrines.
+Remember, your posts should have original content in them. Quotes should not exceed more than half of a publication. Include your own thoughts, photographs, art, and ideas. 
```

Comment:
Steemit uses the word "post" everywhere, not "publication".

Reply:
I would not remove this part, as it gives a fairly clear message based on legally defined terms:
_Limited, sourced material sharing is OK under fair use and fair dealing doctrines._

Reply:
_Quotes should not exceed more than half of a publication._

That seems subjective. What if someone wants to publish a quotes list?

Reply:
They'll get flagged for copy/paste without original content in the post.  It's an ongoing issue.

Reply:
I understand that non-original copy/paste content will get downvoted. That seems valid and we should communicate it.  I don’t think that the “half of the post should be non copied” rule is the right way to say it though.

Reply:
What if we go with "Quotes should not compose an entire publication."

Reply:
Sounds ok to me. I’d also suggest adding “fair use and fair dealing doctrines“ back in too though.